CMakeLists.txt

```
@@ -106,4 +107,10 @@ if(JSONTOOLKIT_TESTS)
       add_subdirectory(test/packaging)
     endif()
   endif()
+
+  if(JSONTOOLKIT_BENCHMARKS)
+    find_package(benchmark REQUIRED)
```

I would have expected this to be:

```suggestion
    find_package(GoogleBenchmark REQUIRED)
```